webapp/package.json

```
@@ -5,36 +5,37 @@
   "author": "Mattermost",
   "license": "Apache-2.0",
   "main": "src/index.js",
+  "private": true,
   "scripts": {
     "build": "webpack --mode=production",
     "lint": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet",
     "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix"
   },
   "dependencies": {
-    "core-js": "3.1.4",
-    "mattermost-redux": "github:mattermost/mattermost-redux#b56e7e3a6f7f5de281fd3541eefec9345a8bbc89",
-    "prop-types": "15.7.2",
-    "react": "16.8.6",
-    "react-intl": "2.9.0",
-    "react-redux": "5.0.7",
-    "redux": "4.0.1"
+    "core-js": "^3.6.5",
```

We generally use exact dep versions in `package.json`. I'm up for broad discussion on this via Mattermost or dev meeting.

```suggestion
    "core-js": "3.6.5",
```

---

Fixed versions require in general manual audit fixes. I think it makes more sense to say "it should work with anything over this". But yes, we can kick up that conversation.

---

Queued an item for the dev meeting https://community-daily.mattermost.com/core/pl/hkzq6sunwt8gpr4n14fd6y45bw